tests/sdk/test_add.py

```diff
@@ -110,10 +109,28 @@ def test_add_data_sample_already_exists(client, data_sample_query, mocker):
     assert m.is_called()
 
 
+def test_add_data_sample_with_paths(client, data_samples_query, mocker):
+    server_response = [{"key": "42"}]
+    m = mock_requests(mocker, "post", response=server_response)
+    with pytest.raises(ValueError):
+        client.add_data_sample(data_samples_query)
+
+    assert m.is_called()
+
+
 def test_add_data_samples(client, data_samples_query, mocker):
     server_response = [{"key": "42"}]
     m = mock_requests(mocker, "post", response=server_response)
     response = client.add_data_samples(data_samples_query)
 
     assert response == server_response
     assert m.is_called()
+
+
+def test_add_data_samples_with_path(client, data_sample_query, mocker):
+    server_response = [{"key": "42"}]
+    m = mock_requests(mocker, "post", response=server_response)
+    with pytest.raises(ValueError):
+        client.add_data_samples(data_sample_query)
+
+    assert m.is_called()
```

Comment:
Almost same comment as above: could you add a comment explaining what you're testing exactly and could you double check that m is called?